14/umbraco-cms/fundamentals/backoffice/sections.md

```diff
@@ -6,30 +6,32 @@ description: >-
 
 # Sections
 
-A section in Umbraco is where you perform specific tasks related to a particular area of Umbraco. For example Content, Settings and Users are all sections. You can navigate between the different sections by clicking the corresponding icon in the section menu which is positioned at the top of the Backoffice.
+A section in Umbraco is where you perform specific tasks related to a particular area of Umbraco. For example Content, Settings, and Users are all sections. You can navigate between the different sections by clicking the corresponding icon in the section menu which is positioned at the top of the Backoffice.
 
 _The **Section menu** is the horizontal menu located at the top of the Umbraco Backoffice._
 
-<figure><img src="images/sections-highlight-sections2.png" alt=""><figcaption></figcaption></figure>
+![Section](images/highlight-sections-v14.png)
 
-
-There are seven default sections that come with Umbraco:
+Umbraco comes with the following default sections:
 
 ## Content
 
-The Content section contains the content of the website. Content is displayed as nodes in the content tree. Nodes can also show content state:
+The Content section contains the content of the website. Content is displayed as nodes in the content tree.
+
+Nodes in Umbraco can display the following content states:
 
-* Grayed out nodes have not been published
-* <img src="images/has-unpublished-version.svg" alt="" data-size="line"> Nodes have unpublished versions (but are currently published)
-* <img src="images/protected.svg" alt="" data-size="line"> Nodes are protected from the public (can only be accessed by a logged in Member)
-* <img src="images/locked.svg" alt="" data-size="line"> Nodes are currently locked/non-deletable
-* <img src="images/is-container.svg" alt="" data-size="line"> Nodes are containers (a List View for example)
```

Comment:
This is still a state that a piece of content can be in, right?

Reply:
No, the content node doesn't have different states/symbols on it anymore.

Image for reference: 
![image](https://github.com/umbraco/UmbracoDocs/assets/82437098/a9709ba8-df39-46a7-89f2-38a93cdb137a)